src/ngMock/angular-mocks.js

```diff
@@ -2072,13 +2072,13 @@ function MockXhr() {
     var header = this.$$respHeaders[name];
     if (header) return header;
 
-    name = angular.lowercase(name);
+    name = angular.$$lowercase(name);
```

Comment:
jQuery doesn't have any logic like that in its lowercasing and we haven't heard any complaints related to Turkish keyboards. Given jQuery's popularity I'd assume this code is no longer needed, especially that ES5 mandates no such differences so I doubt any relatively recent browser would still have whatever issue existed (if it ever has).